docs/rfc/rfc-1.adoc

```diff
@@ -215,7 +215,7 @@ Governable Parameters
   will trigger an early sweep on a wallet.
 - `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
   be considered for a sweep.
-- `base_sweeping_fee_max`: The highest amount of BTC that operators can
+- `base_btc_fee_max`: The highest amount of BTC that operators can
   initially propose as a fee for miners in a sweeping transaction.
```

Comment:
```suggestion
  initially propose as a fee for miners of Bitcoin transaction.
```

Reply:
We are in the Governable Parameters of Sweeping section, I intentionally left it this way so that we are more specific the transaction we are talking about is the sweeping transaction.

Reply:
I see, makes sense.